src/app/store.ts

```diff
@@ -2,5 +1,14 @@
+import { configureStore } from '@reduxjs/toolkit';
+import { combineReducers } from '@reduxjs/toolkit';
+import todosReducer from '../features/todos';
+import filterReducer from '../features/filter';
+import currentTodoReducer from '../features/currentTodo';
```

Comment:
Ensure that the 'currentTodoReducer' is correctly implemented in the '../features/currentTodo' file and is functioning as expected.